mlir/lib/Dialect/QCO/IR/QCOOps.cpp

```
@@ -10,6 +10,14 @@
 
 #include "mlir/Dialect/QCO/IR/QCODialect.h" // IWYU pragma: associated
 
+#include <mlir/IR/Block.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)